packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4386,18 +4388,19 @@ class Flexible extends ParentDataWidget<Flex> {
 ///        child: Column(
 ///         children: <Widget>[
 ///           Container(
-///             color: Colors.red,
+///             color: Colors.amber,
```

Comment:
In the diagram this is blue
```suggestion
///             color: Colors.blue,
```